django/contrib/auth/password_validation.py

```diff
@@ -167,9 +167,14 @@ class CommonPasswordValidator:
     https://gist.github.com/roycewilliams/281ce539915a947a23db17137d91aeb7
     The password list must be lowercased to match the comparison in validate().
     """
-    DEFAULT_PASSWORD_LIST_PATH = Path(__file__).resolve().parent / 'common-passwords.txt.gz'
+
+    @functools.cached_property
```

Comment:
Please use `cached_property` from `django.utils.functional`.

Reply:
I will do this.

However, please be aware that this line of code has become something of a football among reviewers. https://github.com/django/django/pull/13841#discussion_r551560566 asked for `django.utils.functional`. Then https://github.com/django/django/pull/13841#discussion_r570091753 asked for `functools`. And now we're back to `django.utils.functional`. I personally don't care—as I said, I'll make the requested change.

Reply:
Done: 6fe617988edc1326511ca0840b65f9c2e9a2a6fb

Reply:
ticket-30949 is currently not accepted due to some performance issues, so we should use `cached_property` from `django.utils.functional`. We can swap all uses of `cached_property` when implementing ticket-30949.